browser/brave_screenshots/BUILD.gn

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/browser/brave_screenshots/buildflags/buildflags.gni")
+
+assert(enable_brave_screenshots)
+
+source_set("utils") {
+  sources = [
+    "screenshots_utils.cc",
+    "screenshots_utils.h",
+  ]
+
+  deps = [
+    "//chrome/browser/image_editor",
+    "//chrome/browser/ui",
+    "//chrome/browser/ui/browser_window",
+    "//ui/base/clipboard",
+  ]
+}
+
+source_set("brave_screenshots") {
+  sources = [ "brave_screenshots_tab_feature.h" ]
+
+  public_deps = [ "//base" ]
+}
+
+source_set("brave_screenshots_impl") {
```

Comment:
is this interface/impl split still necessary after removing Browser?

Reply:
Tried to do away with it just now, but immediately met a circular dependency. Granted, I may have tried combining the wrong sources.